packages/docs/how-it-works.md

```diff
@@ -9,108 +9,69 @@ icon: light-bulb
-
-In the case that a value is an object (as opposed to an array or a basic type), the Bitcoin Computer will add an extra property `_owners` to the object that contains the list of public keys that are owners. Conversely, if an object is created with a property `_owners` that is set to an array of string encoded public keys, the Bitcoin Computer will arrange the transaction so that the Bitcoin script of the output that represents that object can only be spent by the users that control the public keys in the array.
-
-## Creating Objects and Object Identity
-
-One advantage of associating values with transaction outputs is that the transaction id and output number can be used as an identity for the object. Specifically, whenever an object is created (as in a new memory is allocated) we assign the transaction id and output number to a property called `_id`. The id cannot be reassigned and remains immutable throughout the lifetime of the object.
-
-## Updating Objects and Object Revisions
-
-Whenever a smart object is updated, it's new state is stored in a new output. The transaction id and output number is assigned to a property `_rev` of the object. As any two Bitcoin transactions have distinct ids, each new version of a smart object has a fresh revision.
-
-## Roots and Families of Objects
-
-When a new object is created in an expression that is either a constructor call (that is of the form `new C(...)`) or a function call (that is of the form `x.f(...)`) we can keep track of this occurrence. Specifically, the root of an object is it's id if it is not created within a constructor or function call. If it is created in a constructor call it's root is the id of the newly created object. If an object `y` is created within a function call `x.f(...)` the the root of `y` is the root of `x`.
+## Detailed Description
 
-<!-- 
-## Example
+### Smart Contracts and Objects
 
-Consider a transaction that has the expression `1+2` inscribed. The Bitcoin Computer will associate the value `3` to it's first output. If we spend this transaction with a transaction that is inscribed with the expression `x+4`.
+We will refer to a Javascript expression that is inscribed in a transaction as a *smart contract*. The value that such an expression evaluates to is called a *smart value*. If a smart value is of object type we refer to it as a *smart object*.
 
+### Data Ownership
 
-## Storing Values
+The method of associating data values to outputs described above gives rise to a natural notion of data ownership: A smart value that s *owned* by the users that can spend that output. This is analogous to how satoshis are owned by the users that can spend the output that store the satoshis.
```

Comment:
typo: "A smart value that **is** owned..."

Can we simplify the last sentence?: "This is analogous to how unspent satoshis are owned by the users"